server/src/main/java/org/candlepin/model/Status.java

```diff
@@ -37,7 +37,7 @@
     private boolean standalone;
     private Date timeUTC;
     private String[] managerCapabilities = {"cores", "ram", "instance_multiplier",
-        "derived_product", "cert_v3", "guest_limit", "vcpu"};
+        "derived_product", "cert_v3", "guest_limit", "vcpu", "async_virt_who"};
```

Comment:
Lets not box ourselves in on virt-who, that's a specific client, but we're offering an API for whatever client. Suggest "hypervisors_async"?

What do others think about this one? This is a departure from what we'd normally do which is add a new entry in GET / for a supported resource, similar to "guestids" there now. However it's a little weird in this case, not sure what the "rel" name would be as "hypervisors" is already there.

I'm leaning towards using the capability.